gtsfm/data_association/data_assoc.py

```diff
@@ -121,7 +130,15 @@ def run(
             else:
                 per_rejected_track_avg_errors.append(avg_track_reproj_error)
 
-        track_cheirality_failure_ratio = num_tracks_w_cheirality_exceptions / len(tracks_2d)
+        # aggregate the exit codes
+        triangulation_exit_codes_distribution = Counter(triangulation_exit_codes)
```

Comment:
This can be computed where you compute triangulation_exit_codes, this is also fine though.

Reply:
I dont want to manually count all the exit codes in the loop. This is cleaner